docs/conda.rst

```diff
@@ -0,0 +1,275 @@
+4. Horovod also support that `Gloo`_ collective communications library that can be used in place of 
+   MPI. Include :code:`cmake` in order to insure that the Horovod extensions for Gloo are built.
+
+Below are the core required dependencies. The complete :code:`environment.yml` file is available 
+on GitHub. ::
+
+    dependencies:
+    - bokeh=1.4
+    - cmake=3.16 # insures that Gloo library extensions will be built
+    - cudnn=7.6
+    - cupti=10.1
+    - cxx-compiler=1.0 # insures C and C++ compilers are available
+    - jupyterlab=1.2
+    - mpi4py=3.0 # installs cuda-aware openmpi
+    - nccl=2.5
+    - nodejs=13
+    - nvcc_linux-64=10.1 # configures environment to be "cuda-aware"
+    - pip=20.0
+    - pip:
+        - mxnet-cu101mkl==1.6.* # MXNET is installed prior to horovod
+        - -r file:requirements.txt
+    - python=3.7
+    - pytorch=1.5
+    - tensorboard=2.1
+    - tensorflow-gpu=2.1
+    - torchvision=0.6
+
+The :code:`requirements.txt` file
+---------------------------------
+
+The :code:`requirements.txt` file is where all of the :code:`pip` dependencies, including Horovod 
+itself, are listed for installation. In addition to Horovod we typically will also use :code:`pip` 
+to install JupyterLab extensions to enable GPU and CPU resource monitoring via 
+`jupyterlab-nvdashboard`_ and Tensorboard support via `jupyter-tensorboard`_. ::
+
+    horovod==0.19.*
+    jupyterlab-nvdashboard==0.2.*
+    jupyter-tensorboard==0.2.*
+
+    # make sure horovod is re-compiled if environment is re-built
+    --no-binary=horovod
+
+Note the use of the :code:`--no-binary` option at the end of the file. Including this option 
+insures that Horovod will be re-built whenever the Conda environment is re-built.
+
+Building Conda environment
+--------------------------
+
+After adding any necessary dependencies that should be downloaded via Conda to the 
+:code:`environment.yml` file and any dependencies that should be downloaded via :code:`pip` to the 
+:code:`requirements.txt` file you create the Conda environment in a sub-directory :code:`env` of 
+your project directory by running the following commands.
+
+.. code-block:: bash
+
+    $ export ENV_PREFIX=$PWD/env
+    $ export HOROVOD_CUDA_HOME=$CUDA_HOME
+    $ export HOROVOD_NCCL_HOME=$ENV_PREFIX
+    $ export HOROVOD_GPU_OPERATIONS=NCCL
+    $ conda env create --prefix $ENV_PREFIX --file environment.yml --force
+
+By default Horovod will try and build extensions for all detected frameworks. See the 
+documentation on `environment variables`_ for the details on additional environment variables that 
+can be set prior to building Horovod.
+
+Once the new environment has been created you can activate the environment with the following 
+command.
+
+.. code-block:: bash
+
+    $ conda activate $ENV_PREFIX
+
+The :code:`postBuild` file
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you wish to use any JupyterLab extensions included in the :code:`environment.yml` and 
```

Comment:
I recommend using double backquotes (``) for code highlighting

Reply:
Done